brainstorming.md

```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-> **Status**: brainstorming, pre-proposal
+> **Status**: brainstorming, pre-proposal, in active development. Version: `0.1.0-dev`
```

Comment:
My preference: we call this new spec`0.2.0-dev` and we call the old spec `0.1.0-dev` since we already have releases named `0.1.x` for the old spec at
https://github.com/single-cell-data/TileDB-SOMA/releases/tag/0.1.11
https://github.com/TileDB-Inc/tiledbsc/releases

Reply:
I"m happy to do this, but we can not make the mistake of conflating the API / spec version with the implementation version.  They will evolve independently.  So this is a one-time change to burn another version #.

Reply:
Also, the spec had two different names for this function - fixed to use `get_SOMA_version`